commit 0241624144401879c60ce034035a2d164e3ab99e (refs/changes/45/1445/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-02-06T14:25:02-08:00 (2 years, 8 months ago)
    
    MANTA-3143 moray A record ttls could be higher
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/sapi_manifests/registrar/template.in b/sapi_manifests/registrar/template.in
index fc8e746..c6c46ff 100644
--- a/sapi_manifests/registrar/template.in
+++ b/sapi_manifests/registrar/template.in
@@ -11,7 +11,7 @@
       },
       "ttl": 60
     },
-    "ttl": 30,
+    "ttl": 60,
     "ports": [@@PORTS@@]
   },
 
